pyproject.toml

```diff
@@ -138,7 +138,7 @@ commands =
 
 
 [tool.ruff]
-src = ["src", "tests"]
+src = ["src", "tests", "system_tests"]
```

Comment:
Nit: I thought we were going with the same structure as Hyperion

- tests
     - unit_tests
     - system_tests

I think that might be neater at the top level, but I'm not especially fussed.

Reply:
As long as the default set of directories for `pytest` excludes the `system_tests` directory I'm not too fussed which of these structures we go with.

I'd like to continue running `pytest` for just unit tests, then `pytest /path/to/system_tests` as a separate CI job for the system tests

Reply:
Yes, I did it like this because it gives the behaviour @coretl describes, which I like. I will see how messy it is to have the same behaviour with the nested structure

Reply:
Unless you now have stronger opinions I think leave like this as it gives the cleanest way to only run some of the tests. We can change another time if we want to,